src/Testcontainers.Elasticsearch/ElasticsearchBuilder.cs

```diff
@@ -74,7 +74,7 @@ protected override ElasticsearchBuilder Init()
             .WithEnvironment("discovery.type", "single-node")
             .WithEnvironment("ingest.geoip.downloader.enabled", "false")
             .WithResourceMapping(DefaultMemoryVmOption, ElasticsearchDefaultMemoryVmOptionFilePath)
-            .WithWaitStrategy(Wait.ForUnixContainer().AddCustomWaitStrategy(new WaitUntil()));
+            .WithWaitStrategy(Wait.ForUnixContainer().UntilMessageIsLogged("\"message\":\\s?\"started\"?"));
```

Comment:
Here, I favor the previous implementation. For me, it is a bit more explicit that different log messages are required to detect the readiness of the service (due to different image versions).